examples/3d/pbr_debug.rs

```diff
@@ -0,0 +1,103 @@
+//! Loads and renders a glTF file as a scene and displays rendering debug visualizations.
```

Comment:
Since this is added in the scene_viewer, does this really need a separate example?

Reply:
Maybe not. I wrote pbr_debug first, and then thought "oh... I could add it into `scene_viewer`!"

Reply:
I think I'll remove it.